src/pages/CropSidebar/PlantHardinessZone/PlantHardinessZone.js

```diff
@@ -12,43 +19,85 @@ const PlantHardinessZone = ({
       labelId="plant-hardiness-zone-dropdown-select"
       id="plant-hardiness-zone-dropdown-select"
+      label="ZONE"
       style={{
         width: '100%',
         textAlign: 'left',
       }}
-      sx={{ minWidth: 200 }}
+      sx={{
+        fontWeight: 'bold',
+        minWidth: 200,
+        '& .MuiOutlinedInput-notchedOutline': {
+          borderColor: '#598445', // Set your custom color for the notched outline
+          borderWidth: '2px', // Set your custom border width
+          borderRadius: '4px', // Custom border radius
+        },
+        '&:hover .MuiOutlinedInput-notchedOutline': {
+          borderColor: '#598445', // Custom color on hover
+          borderWidth: '2.5px',
+        },
+        '&.Mui-focused .MuiOutlinedInput-notchedOutline': {
+          borderColor: '#598445', // Custom color when the Select is focused
+          borderWidth: '2.5px', // Custom border width when the Select is focused
+        },
+      }}
+      MenuProps={menuProps}
       onChange={(e) => setRegionShorthand(e.target.value)}
       value={regionShorthand || ''}
       error={!regionShorthand}
     >
-
-      {regionsRedux?.length > 0 && regionsRedux.map((region, i) => (
-        <MenuItem value={region.shorthand} key={`Region${region}${i}`}>
-          {councilLabelRedux !== 'Midwest Cover Crops Council' ? `Zone ${region.shorthand?.toUpperCase()}` : `${region.shorthand?.toUpperCase()}`}
-        </MenuItem>
-      ))}
+      {regionsRedux?.length > 0
+        && regionsRedux.map((region, i) => (
+          <MenuItem value={region.shorthand} key={`Region${region}${i}`}>
+            {councilLabelRedux !== 'Midwest Cover Crops Council'
+              ? `Zone ${region.shorthand?.toUpperCase()}`
+              : `${region.shorthand?.toUpperCase()}`}
+          </MenuItem>
+        ))}
     </Select>
   );
 
   return (
     <Collapse in={regionToggleRedux}>
       <List component="div">
-        <ListItem component="div">
-          <FormControl
-            variant="filled"
-          >
-            <InputLabel>{councilLabelRedux === 'Midwest Cover Crops Council' ? 'COUNTY' : 'ZONE'}</InputLabel>
+        <ListItem component="div" style={{ padding: 0 }}>
+          <FormControl>
+            <InputLabel id="plant-hardiness-zone-dropdown-label" sx={{ fontWeight: 'medium' }}>
+              {councilLabelRedux === 'Midwest Cover Crops Council' ? 'COUNTY' : 'ZONE'}
+            </InputLabel>
             {plantHardinessZone()}
-            {!regionShorthand
-            && (
-            <Typography variant="body2" align="center" color="error" gutterBottom>
-              {`Please Select a ${councilLabelRedux === 'Midwest Cover Crops Council' ? 'County' : 'Zone'}`}
-            </Typography>
+            {!regionShorthand && (
+              <Typography variant="body2" align="center" color="error" gutterBottom>
+                {`Please Select a ${
+                  councilLabelRedux === 'Midwest Cover Crops Council' ? 'County' : 'Zone'
```

Comment:
you should use council shorthand here to avoid breaking if the label changes

Reply:
fixed
- PlantHardinessZone now uses councilShortHand everywhere.